src/main/java/edu/kis/powp/jobs2d/command/manager/CommandManager.java

```diff
@@ -71,19 +73,7 @@ public synchronized DriverCommand getCurrentCommand() {
     }
 
     public synchronized String getStatisticCommand() {
-        if (currentCommand != null) {
-            CommandStatisticVisitor statisticVisitor = new CommandStatisticVisitor();
-            currentCommand.accept(statisticVisitor);
-
-            int totalSubCommandsCount = statisticVisitor.getOperateToCount() + statisticVisitor.getSetPositionCount();
-            int operateToCommandsCount = statisticVisitor.getOperateToCount();
-            double totalCommandLength = statisticVisitor.getTotalLength();
-
-            return "Number of sub-commands used: " + totalSubCommandsCount + "\nNumber of operateTo commands used: " +
-                    operateToCommandsCount + "\nTotal command length: " + totalCommandLength;
-        } else {
-            return "No statistics to display";
-        }
+        return visitorStatisticsGenerator.generateStatistics(currentCommand);
     }
```

Comment:
OK, but this method is not necessary here. Clients can use VisitorStatisticsGenerator directly.